README.md

```diff
@@ -4,17 +4,24 @@ riscv-tests
 About
 -----------
 
-This repository hosts unit tests for RISC-V processors.
+This repository hosts unit tests for RISC-V processors including the instructions of the Xpulpimg extension subsets as defined [here](https://iis-people.ee.ethz.ch/~balasr/pulp-ext/pulp.html).
+
+This repository can be used to do behavioral [Spike](https://github.com/pulp-platform/riscv-isa-sim) simulations of instructions. This allows verification of new instructions or to find out the validity of a test case.
+
+The tests can also be used for RTL simulation (outside of the scope of
+`riscv-tests`).
+
 
 Building from repository
 -----------------------------
 
 We assume that the RISCV environment variable is set to the RISC-V tools
 install path, and that the riscv-gnu-toolchain package is installed.
 
+For 32-bit architectures use the flag `--with-xlen=32`.
```

Comment:
Maybe you want to write this after the bash snippet, and specify you are referring to the `configure` step?